js/globals.ts

```diff
@@ -37,11 +43,10 @@ window.libdeno = null;
 
 // import "./url";
 
-// import * as timer from "./timers";
-// window["setTimeout"] = timer.setTimeout;
-// window["setInterval"] = timer.setInterval;
-// window["clearTimeout"] = timer.clearTimer;
-// window["clearInterval"] = timer.clearTimer;
+window["setTimeout"] = timers.setTimeout;
```

Comment:
I don't really understand why this needs to be `window["foo"]` instead of `window.foo`.
If there's reason maybe add a comment? I'm a bit surprised that prettier leaves it as-is.

Reply:
Fixed.